auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -153,23 +150,15 @@ public void onSaveInstanceState(Bundle outState) {
     @Override
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
-        switch (requestCode) {
-            case RC_HINT:
-                if (data != null) {
-                    mLastCredential = data.getParcelableExtra(Credential.EXTRA_KEY);
-                    if (mLastCredential != null) {
-                        // Get the email from the credential
-                        mEmailEditText.setText(mLastCredential.getId());
-
-                        // Attempt to proceed
-                        validateAndProceed();
-                    }
-                }
-                break;
-            case RC_SIGN_IN:
-            case RC_WELCOME_BACK_IDP:
-                finish(resultCode, data);
-                break;
+        if (requestCode == RequestCodes.CRED_HINT && data != null) {
```

Comment:
So what happened to RC_SIGN_IN and RC_WELCOME_BACK_IDP?  I see we also dropped the handling for those in EmailActivity.

Reply:
Yeah, they've been [squished](https://github.com/firebase/FirebaseUI-Android/pull/1189/files/b1f235685dfe9135d0e51db999b80e8e21e83624#diff-958f7de5a044b2510b5bc0740b2f065bR86) into the `RequestCodes` class for our sanity's sake. So `RC_SIGN_IN` -> `RequestCodes.WELCOME_BACK_EMAIL_FLOW` and `RC_WELCOME_BACK_IDP` ->  `RequestCodes.WELCOME_BACK_IDP_FLOW`.